worlds/Files.py

```diff
@@ -158,6 +158,7 @@ def get_manifest(self) -> Dict[str, Any]:
 class APPlayerContainer(APContainer):
```

Comment:
What's the reason for this PR? Is there a problem we run into that this fixes? Does it enable people to do something they couldn't do before?